tf_quant_finance/black_scholes/approximations/american_option.py

```diff
@@ -83,7 +86,7 @@ def adesi_whaley(*,
       where r are the discount rates, or risk free rates.
       Default value: `None`, equivalent to r = 0 and discount factors = 1 when
       discount_factors also not given.
-    continuous_dividends: An optional real `Tensor` of same dtype as the
+    dividend_rates: An optional real `Tensor` of same dtype as the
       `volatilities`. The continuous dividend rate on the underliers. May be
```

Comment:
Could you please add `continuous_dividends ` to the docstring as well:

continuous_dividends:  `Tensor` equivalent to `dividend_rates`, to be deprecated. 

Here and elsewhere

Reply:
Sure, it does seem to be a good idea to still reference `continuous_dividends` in the doc strings.